### DISCUSSION TITLE
Extract filenames from dataframe

### DISCUSSION THREAD

**Discussion by zane007bloom** [aws/aws-sdk-pandas#860] [Q&A]
Hi,

If I read a bunch of json files from an s3 folder, something like this

`df = wr.s3.read_json(path='s3://mybucket/data/data-wrangler/', dataset=True)`

Is it possible to get the filenames of the files that were read in to create the dataframe?

**Comment by jaidisido** [comment]
[wr.s3.list_objects](https://aws-data-wrangler.readthedocs.io/en/stable/stubs/awswrangler.s3.list_objects.html#awswrangler.s3.list_objects) is the method to use for that effect
